telegram/ext/filters.py

```diff
@@ -192,12 +195,118 @@ def filter(self, message):
     class _Document(BaseFilter):
         name = 'Filters.document'
 
+        class Category(BaseFilter):
+            """This Filter filters documents by their category in the mime-type attribute
+
+            Note:
+                This Filter only filters by the mime_type of the document,
+                    it doesn't check the validity of the document.
+                The user can manipulate the mime-type of a message and
+                    send media with wrong types that don't fit to this handler.
+
+            Examples:
+                Filters.documents.Category("audio/") returnes `True` for all types
+                of audio sent as file, for example "audio/mpeg" or "audio/x-wav"
+            """
+            name = "Filters.document.Category()"
```

Comment:
Please do something like
`self.name = 'Filters.document.category({})'.format(self.category)` in `__init__` instead

Reply:
Yes, that makes sense. I will fix it.